crates/sui-move/src/new.rs

```diff
@@ -34,7 +34,12 @@ impl New {
             r#"/*
 /// Module: {name}
 module {name}::{name};
-*/"#,
+*/
+
+// The following recommendations are based on 2024 Move.
```

Comment:
I would suggest changing the text to something like "For naming coding conventions for Move, see <link>"